.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
### Optional
- [ ] It would be better if we add files(.vscode) that are not related to the project to the `.gitignore` file. This [article](https://code.likeagirl.io/how-to-get-rid-of-ds-store-and-node-modules-in-git-repositories-d37b8a391247) can helps us get started

Reply:
Done, thanks.